libraries/AP_Camera/AP_Camera_MAVLinkCamV2.cpp

```diff
@@ -123,36 +123,6 @@ SetFocusResult AP_Camera_MAVLinkCamV2::set_focus(FocusType focus_type, float foc
     return SetFocusResult::ACCEPTED;
 }
 
-// handle incoming mavlink message including CAMERA_INFORMATION
```

Comment:
I think we should not remove this method.  I think the backend should catch it first and then forward it on